blamo/lib/Test/test.dart

```diff
@@ -47,35 +47,133 @@ class _TestPageState extends State<TestPage> {
                 },
                 autovalidate: true,
                 child: Column(
-                children: <Widget>[ 
+                children: <Widget>[
                   FormBuilderTextField(
-                    attribute: 'type-no',
+                    attribute: 'beginTestDepth',
+                    validators: [FormBuilderValidators.numeric()],
+                    decoration: InputDecoration(labelText: "Begin Test Depth (m)"),
+                  ),
+                  FormBuilderTextField(
+                      attribute: 'endTestDepth',
+                      validators: [FormBuilderValidators.numeric()],
+                      decoration: InputDecoration(labelText: "End Test Depth (m)")
+                  ),
+                  FormBuilderTextField(
+                    attribute: 'soilType',
+                    validators: [],
+                    decoration: InputDecoration(labelText: "Soil Type"), //ASK - preferred title?
+                  ),
+                  FormBuilderTextField(
```

Comment:
Test description should have the same description tag implementation as in units.